internal/app.go

```diff
@@ -184,14 +179,18 @@ func NewApp(sota_config, secrets_dir string, unsafeHandlers, testing bool) (*App
 
 	url := os.Getenv("CONFIG_URL")
 	if len(url) == 0 {
-		url = sota.GetDefault("tls.server", "https://ota-lite.foundries.io:8443").(string)
+		url = sota.GetDefault("tls.server", "https://ota-lite.foundries.io:8443")
 		url += "/config"
 	}
 
+	storagePath := sota.GetOrDie("storage.path")
```

Comment:
Do we need to verify anywhere if this path/location actually exists and is writable? Since we before this actually checked if `SOTA_DIR` was a directory.

Reply:
Good question. I don't think we were checking before, but still - what's the best way to fail? Here's some thoughts:

* If this path doesn't exist - aklite won't be able function.
* This is something that would happen *before* going to production
* Fioconfig is still somewhat functional even without this directory

Given those points - Its probably worth just trying to run and maybe someone can trigger an OTA via fioconfig that could fix the device

Reply:
Maybe somewhere just notify it? Either a WARN or something else that makes a developer/person go "Oops", then fix it in the flow of going to production. :+1: